package.json

```diff
@@ -48,8 +48,10 @@
     ]
   },
   "dependencies": {
+    "@fortawesome/fontawesome-free": "^5.12.1",
```

Comment:
Was this intentionally added?

Reply:
Yes, it's where the sprites seem to live, are they somewhere else? I figured a zero install experience for free icons was better than documenting a separate install step.

Reply:
Nope, they live there and that's the correct place to get them from. But this would force people who are just using Pro to download the extra package. I also worry about someone who is trying to use an older version for one reason or another (we have those folks who aren't ready to upgrade) This is going to add a competing version of the package that might have to contend with hoisting.

Reply:
Makes sense, I'll replace with installation instructions.